packages/gatsby-recipes/rollup.config.js

```diff
@@ -23,38 +24,104 @@ function excludeDevTools() {
   }
 }
 
-export default {
-  input: "src/index.js",
-  output: {
-    file: "dist/index.js",
-    format: "cjs",
+export default [
+  {
+    input: {
+      index: `src/index.js`,
+      "graphql-server/server": `src/graphql-server/server.js`
+    },
+    output: {
+      dir: `dist`,
+      entryFileNames: `[name].js`,
+      format: "cjs",
+      sourcemap: true,
```

Comment:
do we want this to be true?

Reply:
Is there any downside in doing so? 

When bundling things gets tricker to track down to source code, so I did hope that sourcemaps would help in those cases

Reply:
just checking, no downside. slightly slower but that doesn't matter